internal/provider/provider.go

```diff
@@ -149,7 +150,7 @@ func (p *PlanetScaleProvider) Configure(ctx context.Context, req provider.Config
 	// Adding this to `resp.Diagnostics` ensures it will be printed during typical
 	// terraform operations, whereas logging with `tflog.Warn()` will only show if the
 	// users specifies the `TF_LOG` env var.
-	if serviceTokenName != "" && serviceTokenID == "" {
+	if serviceTokenName != "" {
```

Comment:
decided to always show the user the warning if they're using `_NAME`. Previously we would not show the warning if they were specifying both _NAME and _ID, but I figure it might be good to let them know that while they're using _ID they still have _NAME set unncessarily